spring-cloud-gcp-starters/spring-cloud-gcp-starter-config/README.adoc

```diff
@@ -1,4 +1,4 @@
-= Spring Cloud GCP Config Client Starter
+== Spring Cloud GCP Config Client Starter
```

Comment:
Why this README file is still in the `starter` module?
Nobody is going to read it since everybody knows that any Spring Boot Starter is just empty jar with conventional pom.

Why just don't go the way to have docs only in `docs` ?

Reply:
+1 to that. I was asking me the same question (about the duplicated doc, that is).